lib/workflow/stores/mongo.js

```diff
@@ -283,8 +284,15 @@ function mongoStoreFactory(waterline, Promise, Constants, Errors, assert, _) {
                 instanceId: 1
             }
         };
+        var promise = waterline.graphobjects.findAndModifyMongo(query, {}, graph, options);
 
-        return waterline.graphobjects.findAndModifyMongo(query, {}, graph, options);
+        if (isNew) {
+            promise.then(function () {
+                waterline.graphobjects.publishRecord('created', graph);
```

Comment:
@benbp I've already integrated the waterline events into the UI. We could make this kind of change to move the graph object events to another queue, but for now I'd prefer to get it working again in the UI.